Developed in-house, not fed back.
The sql_mode values should read from the server and not from the /etc/mysql/5.7/my.cnf file

--- a/support-files/my-default.cnf.sh
+++ b/support-files/my-default.cnf.sh
@@ -28,4 +28,3 @@
 # sort_buffer_size = 2M
 # read_rnd_buffer_size = 2M 
 
-sql_mode=NO_ENGINE_SUBSTITUTION,STRICT_TRANS_TABLES 
